lxd/instance/drivers/driver_qemu_templates.go

```diff
@@ -681,6 +682,7 @@ func qemuHostDrive(opts *qemuHostDriveOpts) []cfgSection {
 			entries: []cfgEntry{
 				{key: "backend", value: "socket"},
 				{key: "path", value: opts.path},
+				{key: "fd", value: opts.fd},
```

Comment:
Wouldn't this write both the `path` and `fd` setting to the final config file? It looks that only one of them should be there at a given time. Otherwise you might still encounter the original bug as the too long socket path is still in there.

Reply:
Actually I should have just removed the line that includes the path because it serves no purpose(probably because in this case the path is empty). This is how the `chardev` section in `qemu.config` looks like.
```
# metrics drive (virtio-fs)
[chardev "lxd_metrics"]
backend = "socket"
fd = "6"
```